### DISCUSSION TITLE
About the theoretical description of a single photon.

### DISCUSSION THREAD

**Discussion by carlosbelcech** [a-cosmology-group/acg#233] [General]
Hello classmates. I begin this discussion because I would like to better understand our current image of a single photon. For instance, to an elemental level in Quantum Mechanics, the single photon is described as a wave packet in function of position-momentum uncertainties. In general, from this framework, the probability amplitudes for the time function of the position as a Gauss' bell function is obtained. But, is there some correlation between the probability amplitudes of position and the intensities of corresponding electric and magnetic fields? I understand that probability amplitudes of position and electromagnetic field intensities must be highly correlated. One question is how to calculate the time-function and the maximum field amplitude of a single photon. Another is about the polarization state. I understand that an elemental photon is circularly polarized (left or right), but also it can be described as linearly polarized. The last question is about the size. Photons must have a defined size, I guess. Can you help me with this?

**Comment by Francois-Zinserling** [comment]
@carlosbelcech I'm not sure at what level you're expecting engagement but here goes a reply from a naive amateur .. and I tend to the classic side, because you cannot trust a system that invented quarks as a solution ...

My current research is toward defining an aether and it's associations. I'll motivate: This arised after my work on push-gravity (originally inspired partly by @expearth =Matt Edward) determined that some form of an aether must exist. From the works of Alfonso Rueda and Bernard Haisch (2005), Patrick Grahn, Arto Annila, Erkki Kolehmainen (March 2018), where the ZPF and coupled photons are prominent, it seems that photons would make a good base to found an aether on. Unlike the others, I expect a very narrow energy level for the 'aether' photons. OK, enough selling.

To understand how coupled photons or the ZPF might work, I had to delve in to the possible properties of photons which might render them undetectable.  We already know that waves can annihilate, so it is simple that 2 photons perfectly out of phase would completely cancel out any energy. But how so with photons?

For a single photon we know Energy $E = hf$ or $E=ℏω$
And for coupled photons we expect opposites, so a photon polarisation of +h and -h would leave the 2 photons with an undetectable energy signature. The total energy $|E=2f|$ is still there, just not observable conventionally.
The +h and -h would equate to your circularly 'left' or 'right' polarised but for a real photon (non-virtual) there are no more degrees of freedom. Linear polarisation maybe refers to virtual photons? You, know, those things that don't actually exist, like .. quarks? 🗡️ 

Photon polarisation is often confused because we see these images of curled photon streams. That would be for a beam of circularly polarised photons. Each individual photon does not 'rotate' in space, as we know we can test this with polarisers. A photon that passes a linear polariser will 100% pass the next exact same oriented linear polariser. So each photon has +h or -h in a specific spacial orientation, and it stays there until disturbed.

You've opened the playground, I've given the merry-go-round the first push, let's see where this goes.

**Reply by carlosbelcech** [reply]
Hello Francois. Thanks for your reply. I'm searching an outline within the framework of standard quantum mechanics. I guess to at very fundamental level.
In relation to aether ideas, I think that these never was discarded by the theoretical physicists. In the sense that it is a substance o field, inherent to space itself. A property of space itself. In any aether frame, the empty space has properties as viscosity, elasticity, field intensity or...and a zero point field?. In General Relativity, the space can expands, bends, sinks or contracts. In QM, the pristine vacuum has energy...of vacuum, and it is full of particles and movement. In the Standard Model of particles, each type of particle is an excitation of a fundamental and omnipresent field anchored to vacuum, and preexisting to matter. In the model of Bing Bang, there are at least a pair of ominpresent fields, as dark energy, inflaton field or Higg's field, whose source would be, the space itself?. Is to say, from my point of view, we never leave the aether ideas, just it changes its wrapping. 
By other hand, the principle of relativity crashes headlong with all these ideas. I mean, you can to use a aether model for to obtain some descriptions of nature (as is see) but I think that in the future these will be definitely discarded. I consider that the initial frame of Special Relativity is the correct, but it was leaved by Einstein and others for to went to the like-aether frames.
In relation to photon anihilation by interference, it is a interesting issue, we was studying this for a time. There are interesting cases. My pre-ellaborated idea is that energy conservation principle is the zero rule and therefore,  from intereference of two or more photons will raise always a energy conservative pattern.
Thank you again.

**Comment by budrap00** [comment]
@carlosbelcech You may have noticed that there are a wide variety of views on offer here and very little "consensus" beyond a shared disbelief in the standard model of cosmology. You should consider that a good thing. In modern cosmology the consensus is the crisis. I'm prefacing my remarks on the photon in this way to make it clear that what follows is my view of the matter, not any kind of official ACG position.

A photon is a wave quanta and a wave quanta is in physics a single-wavelength wave called a soliton. There are no fractional waves observed in physics - no half or quarter waves. There are of course whole waves with half the wavelength of a longer wave but for any given frequency there is a minimum whole wave - if that wave quanta is not there then there is no wave at that frequency. Solitons are observed phenomenon. They are the minimum wave, the wave quantum.

The idea that solitons are "photons", conceived of as being particles, arises in the incoherent mess that is Quantum-Particle Physics Theory which constitutes the other "standard model" of modern theoretical physics. Photons are electromagnetic solitons or wave quanta. They are not particles in the sense that three dimensionally local particles with rest mass like electrons, protons, atoms, billiard balls, etc. can be considered and treated as "particles". 

"Photons" are not particles in any physically meaningful way except when they are emitted or absorbed in their entirety by an emitter/absorber particle like an electron bouncing between the energy shells surrounding an atom. Only a whole wave with the correct amount of energy can be emitted or absorbed under those circumstances.

The reductionist tendency of theoretical physics raises an additional problem with respect to the study of electromagnetic radiation wherein the observed behavior of EMR on classical and cosmological scales is simply reduced to the sum of quantum-scale wave quanta events. The central conceit is that quantum scale phenomenon are the basis of all classical and cosmological scale phenomenon. Such a theoretical approach is belied by the common observation that complex systems are not reducible to the sum of their parts.

An atom is more than the sum of its electron(s) and proton(s), a molecule is more than the sum of its atoms, the solar system is more than the sum of its individual atoms and molecules, a galaxy is more than the sum of its solar systems, a galaxy cluster is more than the sum of its galaxies, and the Cosmos cannot be summed because its extent is unknown and unknowable. Different scales and complexities require different analytical approaches. The search for one-size-fits-all mathematical models produces one-size-fits-none results like the two current standard models.

**Comment by RedshiftDrift** [comment]
@carlosbelcech your description as ~~"the single photon is described as a wave packet as a function of position-momentum uncertainties"~~ is incorrect.  The single photon is a plane wave of indefinite extent, with energy $E=h\nu$, momentum $\vec{p}= \hbar \vec{k}$ and a polarisation $\hat{e}$ (nothing else).

What you call "a wave packet as a function of position-momentum uncertainties" is a **superposition of photons**, which can then describe a real wavepacket with the probability amplitudes for the time function of the position, and the field intensities you need.

Make sure you are up to date with classical electromagnetism (Jackson, *"Classical electrodynamics"*, Wiley 1975) and quantum mechanics (Cohen-Tannoudji, Diu, Laloë, *"Quantum Mechanics"* Wiley 1977).
Then for QED I would recommend Cohen-Tannoudji, Dupont-Roc, and Grynberg, *"Photons and atoms: introduction to quantum electrodynamics."* New York: Wiley, 1997.

This is a copy of a slide taken from my talk at MG17 (below).  **QED on one page!**  *I hope you will attend my practice talk!* Date & time to be announced soon.

![one-page-QED](https://github.com/a-cosmology-group/acg/assets/16748402/64e0979d-9146-417c-ad0d-5cf9f4fb1fc8)

**Reply by carlosbelcech** [reply]
jaja. ok. I will try! Let me know when.

**Reply by budrap00** [reply]
@RedshiftDrift 

>The single photon is a plane wave of indefinite extent...

A plane wave is a mathematical approximation technique it is not a physical thing:

>A true plane wave cannot physically exist, because it would have to fill all space.

--- https://en.wikipedia.org/wiki/Plane_wave

**Reply by carlosbelcech** [reply]
Yes. I kept thinking about this. An infinite wave train is not a good description of a photon. This leaves us in a position to rethink the Uncertainty Principle. For example, in a laser we know almost exactly the energy of the photons. If we disperse the beam, that is, with a prism until we have isolated beams of photons, we should have better information about the temporal function of the photon positions and a better description. Maybe QED will help us. I don't know QED.

**Reply by RedshiftDrift** [reply]
@carlosbelcech and you probably read that by adding a number of plane waves (called the "Fourier transform") one can get a beam of photons that describe exactly the laser beam you are talking about, with a specific energy per photon and a spatially localized beam.

**Reply by RedshiftDrift** [reply]
@carlosbelcech @sahil5d @HanDeBruijn this description might be useful [A.23 Quan­ti­za­tion of ra­di­a­tion](http://web1.eng.famu.fsu.edu/~dommelen/quantum/style_a/qftqem.html)

See what happens when the author tries to find the value of the elec­tric field at a given po­si­tion and time for $i$ photons in the same state, in the section *A.23.4 Prop­er­ties of the ob­serv­able elec­tro­mag­netic field*.

**Reply by sahil5d** [reply]
I have read it. What exactly is brilliant about any of this formulation of QED? There's a cosine function, so something is oscillating, there's a phase shift, like for any pulse, and there are some sqrt(2)'s and 1/sqrt(2)'s and 1/2's, which isn't that groundbreaking. Why do people believe QED is the "jewel of physics"??

**Reply by carlosbelcech** [reply]
Thank you. I will read "A.23 Quan­ti­za­tion of ra­di­a­tion"

**Reply by sahil5d** [reply]
I also scanned the Cohen-Tannoudji, Dupont-Roc, and Grynberg textbook. The textbook builds QED on SR and Maxwell's Retarded Potential Electromagnetism.

SR is debunked (the Sun doesn't go around the Earth, and atmospheric nitrogen molecules don't have the kinetic energy of v=.98c), so QED is debunked. Maxwell's Retarded Potential Electromagnetism is debunked (practically and theoretically, electric attraction is to instantaneous position), so QED is debunked in a second way. (Coulomb's, Ampère's, and Weber's electromagnetism theories are a far more stable foundation.) And QED Renormalization is debunked (by Feynman and Dyson!), so QED is debunked in a third way.

QED hasn't predicted the electron's magnetic moment (see Unzicker's video that shows postdictions sold as predictions, and embarrassingly, postdictions which were wrong). QED also depends on "virtual photons", which are bloatware.

@RedshiftDrift I recommend that you develop the STz redshift mechanism by basing it not on QED but instead on the minimal, necessary physical concepts.

**Reply by RedshiftDrift** [reply]
@sahil5d Minimal physical concepts are insufficient to describe STz.  Every theory in physics has some flaw, but physicists won't go back to the dark ages by using minimal physics, as you recommend.  I understand that you come from the computer sciences and nobody would want to use a flawed compiler to create their apps.  But science is different and QED is not bloatware.

  Instead of looking at the flaws of QED (divergent zero-energy "renormalization", virtual photons, no fundamental value for the electron magnetic moment, etc.) concentrate on what QED is good for: fine structure of atomic energy levels, Planck spectrum without the ultraviolet catastrophe, correct predictions of recoil momentum and Doppler momentum, correct value of the Rydberg constant, correct cross section for the Klein–Nishina formula, and on and on.

  You think you can debunk an entire theory because you flid a flaw in it... wrong!  What about my flawed hammer, would you throw it out because the handle comes off easily?  Then you would have to do woodworking with the "minimal, necessary" Allen key from IKEA!  You won't get very far with that!  I fixed my hammer with a pair of screws and built gardening boxes, a birdhouse, fixed a frame and nailed it on the wall, and fixed the chair I am sitting on.  Therefore, even if the tool is flawed it can be used to do good work.  *It's a matter of having the skills and experience to do it.*  Your "debunking" is totally irrelevant to my work.

  Same thing for QED: I know its flaws and avoid them (there is no virtual photon in the STz process, and no normalization is necessary, the electron magnetic moment is not involved, etc...)  What it adds to "minimal physical concepts" is the quantization of momentum and energy, and the superposition of wavefunctions during the transfer process.  This explains the "momentum diffusion" necessary for the redshift process, ***a result that nobody else has reached because they used minimal physics in their simplifications***.  Each one of these effects have quantitative experimental confirmation, so QED is appropriate for the calculation of STz.

![hammer](https://github.com/a-cosmology-group/acg/assets/16748402/568e32eb-05cb-496e-ae0a-a44d5fedc942)

**Reply by sahil5d** [reply]
> the flaws of QED (divergent zero-energy "renormalization", virtual photons, no fundamental value for the electron magnetic moment, etc.)

We agree on a lot


> concentrate on what QED is good for: fine structure of atomic energy levels, Planck spectrum without the ultraviolet catastrophe, correct predictions of recoil momentum and Doppler momentum, correct value of the Rydberg constant, correct cross section for the Klein–Nishina formula, and on and on.

These achievements are great, but we just established that QED has flaws. Therefore, these achievements should be reconceived without QED.

Like how the Fresnel coefficient, Fizeau experiment, and Airy experiment findings are not exclusive to SR, so the various facts of atomic fine structure and scattering cross sections may not be exclusive to [QED as dependent on SR and Maxwell's RP EM].


> You think you can debunk an entire theory because you [find] a flaw in it... wrong!

Yes. This is standard operating procedure for diagnosing an idea.

Also, the successes of the flawed theory can be factored out and preserved.


> What about my flawed hammer, would you throw it out because the handle comes off easily?

As you are developing a groundbreaking redshift model to challenge and defeat the Big Bang Theory, I recommend not basing it on something which has flaws, and flaws that you are aware of!, when instead you could base it on the specific non-flawed parts of QED that you like. The specific formula, the specific experiments.

There is a theory waiting to be developed that is [quantum electrodynamics without SR and without Maxwell's RP EM].

**Reply by RedshiftDrift** [reply]
@sahil5d life is not long enough to develop a new physical theory **and** a new cosmology.  As I said, STz is based on the specific non-flawed parts of QED.

**Comment by genotics** [comment]
THis from Nature Briefs may add some spice to the stew:

"An observatory in the depths of the Mediterranean Sea has spotted what could be the most energetic neutrino ever detected — “a fantastic event”, says physicist Francis Halzen. Neutrinos are tiny subatomic particles that travel at nearly the speed of light and are thought to come from cataclysmic cosmic events, such as a growth spurt of a supermassive black hole. For now, [the team is staying quiet about the direction the neutrino came from](https://nature.us17.list-manage.com/track/click?u=2c6057c528fdc6f73fa196d9d&id=b320f98b29&e=c09f5819cb) to avoid tipping off competitors about its origin."

Normal physicists would encourage colleagues to verify the discovery.

**Comment by AngelBlanco19** [comment]
@carlosbelcech The first thing you need is a physical model of the real photon (the one that does not fly with light) and to understand its method and way of existence. Independent of the power of infinity of any of the 28 different elementary particles of Matter or elementary UNITS of existence and the photon is the main and most abundant particle being the Universal boson and one of the 4 bosons of Matter, the exact and fixed diameter is always 1•10^-34 m and DOES NOT depend on any unit of measurement that can be invented. Here's more on the subject: «The Gravitational Paradox and the Proposed Experiment to Demonstrate Gravitational Distortion of "Spacetime"» [https://vixra.org/abs/2202.0042](https://vixra.org/abs/2202.0042)

You should also look at this other work to understand something about the physical process of Time, the basis of existence itself: «The Physical Mathematics and Geometry of Dialectical Materialism Versus the Euclidean "Mathematics" and "Geometry" of Philosophical Idealism» [https://vixra.org/abs/2402.0116](https://vixra.org/abs/2402.0116)
![Dibujo de las cosechas  Esquema de la materia y el Universo  Editado  Tamaño reducido](https://github.com/user-attachments/assets/76a5608e-bc9a-48ce-b709-4ff8a059d4e5)
